CitiesUIService/src/main/java/com/example/dto/Cities.java

```diff
@@ -4,8 +4,8 @@
  * Created by kostya.nikitin on 8/31/2016.
  */
 public class Cities {
-    long id;
-    String cityname;
+    public long id;
```

Comment:
Otherwise i get error from org.thymeleaf.exceptions.TemplateProcessingException : Exception evaluating SpringEL expression: "city.cityname" 
I not have public getter for this propertie. Therefore, this error occurs